server/channels/web/handlers.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/mattermost/mattermost/server/v8/platform/services/tracing"
 )
 
+const (
+	frameAncestors = "'self' teams.microsoft.com"
```

Comment:
Hard-coding this seems brittle. Can we make this pluggable or configurable instead?

Reply:
In the interest of time (tight deadline) I'll leave this hard-coded for now, and create a ticket to either add a system console config setting, or create a way for plugins to inject domain names to this list.  I'm not worried about MS Teams changing domains as that would break so much more on their end.

Reply:
Ticket created:  https://mattermost.atlassian.net/browse/MM-53561